app/assets/javascripts/area_chart.js

```diff
@@ -2,32 +2,32 @@ $(function() {
   window.AreaChart = {
     createChart: function(title, data, yAxisTitle, chartType, options) {
       var options = options || {};
-      var container = options['container'] || '#container';
+      var container = Kitechart.getWithDefault(options['container'], '#container');
```

Comment:
Inside the `$(function() {})`, before the `window.AreaChart = ...` you could do 
`var getWithDefault =  Kitechart.getWithDefault;`  to make this a little cleaner.